ide/csl.api/test/unit/src/org/netbeans/modules/csl/api/test/CslTestBase.java

```diff
@@ -3062,8 +3062,12 @@ public boolean isCaseSensitive() {
     }
 
     public void checkCompletionDocumentation(final String file, final String caretLine, final boolean includeModifiers, final String itemPrefix) throws Exception {
+        checkCompletionDocumentation(file, caretLine, includeModifiers, itemPrefix, QueryType.COMPLETION);
+    }
+
+    public void checkCompletionDocumentation(final String file, final String caretLine, final boolean includeModifiers, final String itemPrefix, QueryType queryType) throws Exception {
         // TODO call TestCompilationInfo.setCaretOffset!
-        final QueryType type = QueryType.COMPLETION;
+        final QueryType type = queryType;
```

Comment:
@sdedic I would like to test the cc documentation when the query type is `DOCUMENTATION`. Can we add this test method to `CslTestBase`?

Reply:
Definitely

Reply:
Thank you!